AdblockPlusSafari/AdblockPlus.h

```diff
@@ -35,14 +35,16 @@ typedef NS_ENUM(NSUInteger, AdblockPlusErrorCode) {
 
 @interface AdblockPlus : NSObject
 
+@property (nonatomic, strong, readonly) NSString *__nonnull bundleName;
+
 @property (nonatomic, strong, readonly) NSUserDefaults *__nonnull adblockPlusDetails;
 
 - (NSString *__nonnull)group;
 
-- (NSString *__nonnull)contentBlockerIdentifier;
-
 - (NSString *__nonnull)backgroundSessionConfigurationIdentifier;
 
+- (NSString *__nonnull)contentBlockerIdentifier;
```

Comment:
I can't seem to figure out why this declaration is being moved down - is that related to issue 3184?